patternlab/styleguide/source/_patterns/01-atoms/09-media/figure~dataviz.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think we can remove this file now, right? We just want `figure--dataviz.json` I believe.